GNUmakefile

```diff
@@ -3,7 +3,7 @@ BINARY=packer-plugin-${NAME}
 
 COUNT?=1
 TEST?=$(shell go list ./...)
-HASHICORP_PACKER_PLUGIN_SDK_VERSION?=$(shell go list -m github.com/hashicorp/packer-plugin-sdk | cut -d " " -f2)
+PACKER_SDC = go run github.com/hashicorp/packer-plugin-sdk/cmd/packer-sdc@v0.4.0
```

Comment:
Is there a need to hard-code a specific Packer SDC version here?

Reply:
Sorry, but I don't follow you.

Currently with the `GNUmakefile` on `main` the `HASHICORP_PACKER_PLUGIN_SDK_VERSION` will equate to `v0.5.2` because of this line:

https://github.com/cirruslabs/packer-plugin-tart/blob/ca60a526ce34a783faae7127ad84968143a2b6a8/go.mod#L11

Why should we hardcode it to be `v0.4.0`?

The SDK will change it's version in the future many times anyways, even if it's not that often, and I don't think it's a great idea to patch this by hand each time because currently the @dependabot will update the `go.mod` for us, and this will in turn change the `HASHICORP_PACKER_PLUGIN_SDK_VERSION` accordingly.

Reply:
Ah, you're right. I'll remove this refactoring from the PR, it's not needed :)